packages/commonwealth/client/scripts/state/api/config.ts

```diff
@@ -0,0 +1,13 @@
+import { QueryClient } from '@tanstack/react-query';
+
+export const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      refetchOnWindowFocus: false,
+    },
+  },
+});
+
+export const ApiEndpoints = {
+  BulkTopics: '/bulkTopics',
+};
```

Comment:
this list should be extended by other endpoints (mainly GET, but can be also from other verbs) that will be moved to the react-query. It has two functions:
- we avoid having string endpoints 
- we use it for query-key and this is crucial that query-key is the same for fetching, invalidating and any cache handling

Reply:
how about we upper snake case `BulkTopics` (ex: `BULK_TOPICS`)? just to indicate its a constant. Sometimes we might have to build an endpoint, that we could lowecase/camelcase.